src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj

```diff
@@ -17,6 +17,7 @@
   <Target Name="AddFrameworkFilesToPackage" DependsOnTargets="ResolveLibrariesFromLocalBuild" BeforeTargets="GetFilesToPackage">
     <ItemGroup>
       <ReferencePath Include="@(LibrariesRefAssemblies)" />
+      <ReferencePath ReferenceAssembly="%(Identity)" />
```

Comment:
Where's the metadata used?

Reply:
This is used in the SharedFramework SDK now (as of the arcade update PR that's currently in flight) to select the (usually Roslyn-produced) ref assembly instead of the implementation assembly when building a ref pack.

Reply:
This was changed in https://github.com/dotnet/arcade/pull/15209